02-express-tutorial/app.js

```diff
@@ -70,6 +100,13 @@ app.all("*", (req, res) => {
   res.status(404).send("pg not found!");
 });
 
+/*app.put("/api/people/:id", (req, res) => {
```

Comment:
As you keep building, it may be easier to delete commented out code for readability. Totally fine now, especially if it helps with remembering